doc/man1/genrsa.pod

```diff
@@ -104,9 +104,7 @@ may vary somewhat.
 =head1 BUGS
 
 A quirk of the prime generation algorithm is that it cannot generate small
-primes. Therefore the number of bits should not be less that 64. For typical
-private keys this will not matter because for security reasons they will
-be much larger (typically 1024 bits).
+primes. Therefore the number of bits should not be less than 64.
```

Comment:
I don't see problems with the last sentence of this paragraph...why did you want to remove that? (Though I think it should be adjusted to `typically 2048 bits` instead...)

Reply:
It says that small keys (less than 64 bits) are not a problem because the typical size is 1024 bits. So yes, now you should increase it to 2048 and later maybe even more. I don't think it is required to mention the typical size which may vary. I just try to make it simple.

Reply:
I'm not sure why we have this section at all...  You could add a sentence to the description of *numbits* that says that sizes less than 64 bits are not allowed.

Reply:
> You could add a sentence to the description of numbits that says that sizes less than 64 bits are not allowed.

Seems reasonable...